sqlglot/tokens.py

```diff
@@ -799,6 +800,7 @@ class Tokenizer(metaclass=_Tokenizer):
         "LIMIT": TokenType.LIMIT,
         "LOAD": TokenType.LOAD,
         "LOCK": TokenType.LOCK,
+        "LOCALTIME": TokenType.LOCALTIME,
```

Comment:
We should pop this token off of `KEYWORDS` for BigQuery, ClickHouse, Hive (affects Spark2, Spark, Databricks– which is correct), Oracle, Redshift, T-SQL. Otherwise, we'll incorrectly produce a `Localtime` node instead of a `Column`. See this [comment](https://github.com/tobymao/sqlglot/pull/5982#issuecomment-3346703555).